playbooks/bootstrap-node.yml

```
@@ -0,0 +1,139 @@
+  hosts: "{{ target_host | default('all') }}"
+  tags: [ bootstrap, setup ]
+  become: true
+  vars:
+    sovereign_owner: "dsom-admin"
+    dsom_group: "dsom-admin"
+
+  tasks:
+    - name: "Control Node | Ensure Ansible Controller has SSH Key"
+      ansible.builtin.user:
+        name: "{{ lookup('env', 'USER') }}"
+        generate_ssh_key: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)